src/test/resources/org/opentripplanner/apis/gtfs/queries/vehicle-rental-station.graphql

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Well done for finding out how to use these tests.

Reply:
We are now adding tests for this in two different prs, since this is a more complete version, lets try to get this merged first and I'll update https://github.com/opentripplanner/OpenTripPlanner/pull/5411 afterwards.